apps/runner.sh

```diff
@@ -86,9 +86,14 @@ phone_home_url=$(jq -r .phone_home_url "$metadata")
 trap fail EXIT
 
 service docker start
-until docker info; do
+i=0
+# shellcheck disable=SC2169
+until docker info >/dev/null || [[ $i -gt 5 ]]; do
```

Comment:
I feel like this 'wait for docker to be ready' code that's duplicated should be pulled out into a shared function...  but not strongly enough to block the review on.

Reply:
Would agree but this other file would also have to be fetched from /init (really we should just embed them into the initramfs) which I kind of don't really want to do.